docs/python/py_sdk_docs.md

```diff
@@ -125,6 +133,25 @@ def some_function():
         lighthouse.error(e, email="bob@codelighthouse.io")
 ```
 
+When you're sending errors manually using this method, you can also optionally attach additional data that will show
+up in the admin panel in the error view. The most common use case for this is including additional information that will
+help your developers to identify and debug the error. For example, you could attach information about the currently
+logged-in user that experienced the error, connection information, or other application state information.
+
+```python
+def some_function():
+    try:
+        call_a_broken_function()
+    except NameError as e:
+        lighthouse.error(e, email="bob@codelighthouse.io", data=some_data)
+```
+
+Make sure that the data you're passing (via the `data` argument as show above) can be serialized into JSON. Dictionaries
+and native types will easily work, but if you choose to pass an object, pass the class's `__dict__` property instead 
+and ensure that it does not contain circular references. For more information on the `__dict__` property of Python
+classes, refer to [the python docs](https://docs.python.org/3/library/stdtypes.html#object.__dict__).
```

Comment:
given this info, should we automatically do this for them?

I.E. should we check to see if it works as a serializable object, if not, then we can check to see if it's an object (by trying to pass `object.__dict__` -- if that fails, then we just don't pass it, but we leave a message letting them know that it was non-serializable.

Reply:
yeah you're right we should handle this for them - I'll work on it

Reply:
8df4d0dfcd45b7afe67980e8b644b3067df760d8